### PR TITLE
INF-193 Commenting the SQL scripts

### PR DESCRIPTION
**Pull request by rhosking**
WIP pull request to capture work to comment sql scripts to improve readability

### CODE REVIEW

File: academic_observatory_workflows/database/sql/create_doi.sql.jinja2 [hunk 1]
```diff
@@ -30,37 +32,44 @@ FROM
...
   ARRAY(SELECT as STRUCT identifier, MAX(name) as name, ["Region"] as types, CAST(ARRAY_AGG(home_repo IGNORE NULLS) AS ARRAY<STRING>) as home_repo, CAST(MAX(country) as STRING) as country, CAST(MAX(country_code) as STRING) as country_code, CAST(MAX(country_code_2) as STRING) as country_code_2, CAST(MAX(region) as STRING) as region, CAST(MAX(subregion) as STRING) as subregion, CAST(NULL as STRING) as coordinates, COUNT(*) as count, ARRAY_AGG(DISTINCT member_identifier IGNORE NULLS) as members FROM UNNEST(regions) GROUP BY identifier) as regions,
   ARRAY( SELECT as STRUCT grouping_entity.group_id as identifier, MAX(grouping_entity.group_name) as name, ["Grouping"] as types, CAST([] AS ARRAY<STRING>) as home_repo, CAST(NULL as STRING) as country, MAX(grouping_entity.country_code) as country_code, MAX(grouping_entity.country_code) as country_code_2, CAST(NULL as STRING) as region, CAST(NULL as STRING) as subregion, CAST(NULL as STRING) as coordinates, ARRAY_AGG(DISTINCT grid_group.member_identifier IGNORE NULLS) as members FROM UNNEST(grid_groups) as grid_group, UNNEST(grid_group.groupings) as grouping_entity GROUP BY grouping_entity.group_id) as groupings,
```
Comment (jdddog) [inline academic_observatory_workflows/database/sql/create_doi.sql.jinja2:45]: Perhaps the comments and the SQL could be broken into multiple lines to make it easier to read., e.g. keeping the lines under 120 characters long.

**Review by jdddog — CHANGES_REQUESTED** [review]
The documentation looks good Richard. It would be good to reformat the SQL so that it only takes up 120 characters for each line, so that it is easier to read, if that is possible.

File: academic_observatory_workflows/database/sql/create_doi.sql.jinja2 (hunk 1 above)
Comment (rhosking) [reply academic_observatory_workflows/database/sql/create_doi.sql.jinja2]: not perfectly 120 characters. But I went through and broke up many of the longer lines, and ensured the comments were more consistently styled

**Review by jdddog — APPROVED** [review]
Thanks Richard

**Pull request merged**